scoop-basics/src/main/java/com/example/scoop/basics/scoop/DaggerScreenScooper.java

```diff
@@ -18,10 +22,10 @@
             Object module = controllerModule.value().newInstance();
             screenInjector = parentDagger.extend(module);
         } catch (Throwable e) {
-            throw new RuntimeException("Module not specified for " + screen.getController().getSimpleName(), e);
+            throw new RuntimeException("Module not specified for " + screen.getClass().getSimpleName(), e);
```

Comment:
@itspbj Let's change error message to failed to instantiate module for specified screen